pkg/controller/controlplane/valuesprovider.go

```diff
@@ -655,10 +683,20 @@ func getControlPlaneShootChartValues(
 		return nil, err
 	}
 
+	// get the ca.crt for caBundle of the snapshot-validation webhook
+	secret := &corev1.Secret{}
+	if err := client.Get(ctx, kutil.Key(cluster.ObjectMeta.Name, string(secrets.CACert)), secret); err != nil {
```

Comment:
I guess this is not a proper usage of `secrets.CACert` - to my understanding it rather denotes a CertType in `CertificateSecretConfig` that later gets generated. We rather want to use `SecretNameCACluster` that denotes the `ca` Secret name.

```suggestion
	if err := client.Get(ctx, kutil.Key(cluster.ObjectMeta.Name, v1beta1constants.SecretNameCACluster), secret); err != nil {
```